src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  static addAnimal(obj) {
+    this.alive.push(obj);
+  }
+
+  static killAnimal(obj) {
+    if (!obj.health) {
```

Comment:
Not the best check. It will work incorrectly if somehow obj.health will be below 0.
Filter logic should also be changed.
```suggestion
    if (obj.health <= 0) {
```